content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Background for Public Accessibility
+kind: guide
+further_reading:
+- link: "/security/cspm"
+  tag: "Documentation"
+  text: "Cloud Security Posture Management"
+- link: "/security/default_rules/#cat-cloud-security-management"
+  tag: "Documentation"
+  text: "Out-of-the-box Detection Rules"
+---
+
+Datadog uses a graph processing framework to map relationships between cloud resources to determine whether they are accessible from the internet. This guide outlines the logic used to classify resources as publicly accessible within the graph framework. 
+
+For more information on network reachability, see the [AWS documentation][34] and the [AWS Network Reachability Analyser][35]. Currently, the `Is Publicly Accessible` facet is only available for AWS resources.
+
+## Resource Dependency Graph
+
+The following diagram shows how related resources are used to determine whether other resources are publicly accessible. For example, a Cloudtrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, we will show this relation in the CSPM Resource relationships graph.
```

Comment:
```suggestion
The following diagram shows how related resources are used to determine whether other resources are publicly accessible. For example, a Cloudtrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the CSPM Resource relationships graph.
```